redfish-core/lib/update_service.hpp

```diff
@@ -794,6 +812,12 @@ inline void requestRoutesUpdateService(App& app)
         // Get the MaxImageSizeBytes
         asyncResp->res.jsonValue["MaxImageSizeBytes"] =
             bmcwebHttpReqBodyLimitMb * 1024 * 1024;
+        nlohmann::json& updateSvcConUpdate =
+            asyncResp->res
+                .jsonValue["Actions"]["Oem"]["#OemUpdateService.v1_0_0."
+                                             "ConcurrentUpdate"];
+        updateSvcConUpdate["target"] = "/redfish/v1/UpdateService/Actions/Oem/"
+                                       "OemUpdateService.ConcurrentUpdate";
```

Comment:
Same here, put the URL all in one string as per above comments.

Reply:
Done